CRM/MembershipExtras/Hook/Alter/MemberTabPageContent.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * Alters Memberships Tab of Contact's detail view.
+ */
+class CRM_MembershipExtras_Hook_Alter_MemberTabPageContent {
+
+  public function __construct(&$content) {
+    $this->content = &$content;
+  }
+
+  /**
+   * Executes alterations for the content of the page.
+   */
+  public function alterContent() {
+    $this->appendJSCodeToWatchTotalAmountValueChanges();
+  }
+
+  /**
+   * Appends a '.change()' to throw the event when the total amount is altered.

```

Comment:
the docblock contain and implementation detail (Appends a '.change()')  which is not a good idea (because if you changed the method implementation to not use .change() hen you also should remember to change the docblock (and u will probably forget that and the doblock will be outdated) ... I also think in this case that the method name is clear and no need for the docblock at all so u can remove it.